src/test/scala/molmed/apps/MyLoggers.scala

```diff
@@ -0,0 +1,27 @@
+package molmed.apps
```

Comment:
Yes, it will be. I is a dirty quick test when I was having a headache with log4j and slf4j conflict.